assets/dev/js/editor/views/base-container.js

```diff
@@ -280,17 +280,35 @@ Marionette.CollectionView.prototype.buildChildView = function( child, ChildViewC
  * @inheritDoc
  */
 Marionette.CompositeView.prototype.attachBuffer = function( compositeView, buffer ) {
```

Comment:
This function needed update, because the Mega Menu structure has more levels than the other widgets. 
I have implemented a more general solution that works with all widgets.

Reply:
@hein-obox I don't feel comfortable with these changes in Core right now. I am ok if you done overrides in Pro specifically for the MM.\

I am not sure calling operations like `insertBefore()` and `remove()` on the wrapper fits here in the attachBuffer() method.